src/app/date-picker/date-picker.component.ts

```diff
@@ -23,6 +25,16 @@ export class DatePickerComponent {
     this.previousDate = { year: previous.getFullYear(), month: previous.getMonth(), day: previous.getDate() };
   }
 
+  public setToDate(miliseconds: number) {
```

Comment:
we never pass null. Otherwise it'll show 1970 1st Jan. It sucks. There is no way to empty it out. I'll fix it in the future by forking that guy's git repo and adding the functionality :D Not right now, though